### ISSUE TITLE
The Postman edition tab is desperately empty

### ISSUE THREAD

**Issue by apautrot** [a85/POSTMan-Chrome-Extension#6048]
**Describe the bug**
I'm rather new to postman, haven't used it since month.. but I don't get what I see.
The edition tab is empty. Clicking on the '+' sign does nothing. Clicking on '...' neither.
Take a look at the attached screenshot, I believe you'll understand there is something wrong.

**To Reproduce**
1. Click on '+' sign button in edition tab (main tab of the app)
2. Click on '...' button in edition tab
3. Nothing happen

**Expected behavior**
Something. Anything. Some UI to edit something in my request.

** Screenshot **
![issue](https://user-images.githubusercontent.com/4578723/54083051-fc26ec80-431e-11e9-9dca-12c81d84add0.jpg)

**App information (please complete the following information):**
- Native application
- Postman Version 7.0.4
- OS: Windows 10

**Comment by apautrot** [comment]
I have an update.
I finally found something.
I clicked on `My Workspace` button. I clicked on `Personal`, then on `Create New`.
That refreshed the window and I finally had the request displayed and editable in the main view.
But then I clicked the workspace button to select "My Workspace" again, and now the app is stuck.

![Issue](https://user-images.githubusercontent.com/4578723/54139149-38427600-4421-11e9-9724-e2c58cdabc40.jpg)

Looks like there is a kind of semi-opaque overlay on it.

Note that I did updated from version 6.0.7 to version 6.7.4 to version 7.0.4, the same day.

Is there a way to clean absolutely all config/data/cache and reinstall the app ?
